ocs_ci/helpers/helpers.py

```
@@ -7520,3 +7520,113 @@ def wait_for_osds_down(osd_ids: list[str], timeout: int = 300, sleep: int = 10)
+    current_pools = (
+        sc_data.get("spec", {})
+        .get("managedResources", {})
+        .get("cephFilesystems", {})
+        .get("additionalDataPools", [])
+    )
+
+    new_entry = {
+        "name": pool_name,
+        "erasureCoded": {"dataChunks": data_chunks, "codingChunks": coding_chunks},
+    }
+    updated_pools = current_pools + [new_entry]
```

Ruff (0.15.20)

[warning] Consider `[*current_pools, new_entry]` instead of concatenation

Replace with `[*current_pools, new_entry]`

(RUF005)